app/src/main/java/com/hyperion/ths/marvel_03/ui/main/MainViewPageAdapter.java

```diff
@@ -0,0 +1,59 @@
+package com.hyperion.ths.marvel_03.ui.main;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.view.PagerTabStrip;
```

Comment:
ERROR: Unused import: android.support.v4.view.PagerTabStrip. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports